src/openkit-shared/Providers/IPRNGenerator.cs

```diff
@@ -0,0 +1,40 @@
+﻿//
+// Copyright 2018 Dynatrace LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+namespace Dynatrace.OpenKit.Providers
+{
+    /// <summary>
+    /// Interface providing random numbers
+    /// </summary>
+    public interface IPRNGenerator

```

Comment:
It would be cool if the two methods describe if the boundaries are included or excluded